Form-Controls/index.html

```diff
@@ -6,23 +6,74 @@
   <title>My form exercise</title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&family=Quicksand:wght@300&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
   <header>
-    <h1>Product Pick</h1>
+    <div>
+      <h1>Product Pick</h1>
+    </div>
   </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
+  <section>
+    <div class="section__container">
+      <form action="">
+        <fieldset>
```

Comment:
Does this fieldset make sense? What is the purpose of fieldset in HTML forms?

Reply:
Haha no, it doesn't. I removed it. Its purpose of dividing the content of the form, but I have only one section, hence it is redundant. Thanks